pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/provisionpolicy/policy_rama.go

```diff
@@ -144,3 +172,36 @@ func (p *PolicyRama) sanityCheck() error {
 
 	return errors.NewAggregate(errList)
 }
+
+func (p *PolicyRama) getReclaimStatus() (usage float64, cnt int) {
+	usage = 0
+	cnt = 0
+
+	f := func(podUID string, containerName string, ci *types.ContainerInfo) bool {
+		if ci.QoSLevel != apiconsts.PodAnnotationQoSLevelReclaimedCores {
+			return true
+		}
+
+		containerUsage, err := p.metaServer.GetContainerMetric(podUID, containerName, consts.MetricCPUUsageContainer)
+		if err != nil {
+			containerUsage = ci.CPURequest
+		}
+
+		// FIXME: metric server doesn't support to report cpu usage in numa granularity,
```

Comment:
should we refer to malachite developer for more data about per-container-numa usage?

Reply:
They are already working on offering per cpu usage of each cgroup, but not available now.